src/nvim/msgpack_rpc/channel.c

```diff
@@ -731,101 +535,44 @@ static void unsubscribe(Channel *channel, char *event)
   xfree(event_string);
 }
 
-/// Close the channel streams/process and free the channel resources.
-static void close_channel(Channel *channel)
+
+/// Release the rpc infastructures reference to the channel
+/// Don't call this directly, call channel_close(id, kChannelPartRpc, &error)
```

Comment:
Is this comment still relevant? `channel_process_exit_cb` calls `rpc_close()` directly.

Reply:
The other way around, that call can be removed, as `receive_msgpack` should've been called with `eof` before `exit_cb`